doc/recipe/overview.rst

```diff
@@ -113,22 +113,25 @@ case the two experiments have different ensemble names
 ESMValTool also supports a simplified syntax to add multiple ensemble members from the same dataset.
 In the ensemble key, any element in the form `(x:y)` will be replaced with all numbers from x to y (both inclusive),
 adding a dataset entry for each replacement. For example, to add ensemble members r1i1p1 to r10i1p1
-you can use the following abreviatted syntax:
+you can use the following abbreviated syntax:
 
 .. code-block:: yaml
 
     datasets:
-      - {dataset: CanESM2, project: CMIP5, exp: historical, ensemble: r(1:10)i1p1, start_year: 2001, end_year: 2004}
+      - {dataset: CanESM2, project: CMIP5, exp: historical, ensemble: "r(1:10)i1p1", start_year: 2001, end_year: 2004}
 
 It can be included multiple times in one definition. For example, to generate the datasets definitions
 for the ensemble members r1i1p1 to r5i1p1 and from r1i2p1 to r5i1p1 you can use:
 
 .. code-block:: yaml
 
     datasets:
-      - {dataset: CanESM2, project: CMIP5, exp: historical, ensemble: r(1:5)i(1:2)p1, start_year: 2001, end_year: 2004}
+      - {dataset: CanESM2, project: CMIP5, exp: historical, ensemble: "r(1:5)i(1:2)p1", start_year: 2001, end_year: 2004}
 
 Please, bear in mind that this syntax can only be used in the ensemble tag.
+Also, note that the combination of multiple experiments and ensembles has not been yet supported.
+For example running a recipe with exp: [historical, rcp85], ensemble: [r1i1p1, "r(2:3)i1p1"] may generate a dataset
+that contains only the r2i1p1 ensemble for rcp85, skipping the ensemble r3i1p1.
```

Comment:
The syntax with using a list concatenate the data from all the specified datasets into a single cube, so I don't think it makes sense to use it like this. This was implemented for cases where the names of the ensemble members were different for two `exp`s that you want to have concatenated, see the description here: https://github.com/ESMValGroup/ESMValCore/pull/207

Reply:
I think users can benefit from this additional information because currently, it is possible to make this combination. The esmvaltool doesn’t raise an error in this case and the recipe run is successful. However, users should know that the results are not reliable.

Reply:
I've pushed some changes to this branch so this will raise a RecipeError. Could you try if it works for you? Do you have a suggestion for how the documentation about this could be made more clear?

Reply:
@bouweandela thanks. Good, now it raises an error for that combination. I updated the documentation, accordingly.

Reply:
Thanks!